js/components/Talent.js

```diff
@@ -64,31 +64,47 @@ class BudgetTalent extends HTMLElement {
           width: 100%;
           text-align: center;
           position: relative;
+          top: 25px;
+          font-size: 18px;
         }
         .id {
           width: 100%;
           text-align: center;
           position: relative;
+          top: 18px;
           color: grey;
         }
+         .talent-team-name {
+           width: 100%;
+           text-align: center;
+           position: relative;
+           top: 25px;
+           font-size: 18px;
+         }
+         .talent-team-id {
+           width: 100%;
+           text-align: center;
+           position: relative;
+           top: 20px;
+           color: grey;
+         }
         .budget {
           width: 100%;
           text-align: center;
           position: relative;
-          color: red;
-        }
-        .talent-team-name {
-          width: 100%;
-          text-align: center;
-          position: relative;
+          top: 25px;
+          margin-bottom: 30px;
+          color: #E46A35;
+          font-size: 18px;
         }
+
       </style>
       <img style="margin-bottom: 10px" src="${imgURL}" onerror="this.src='${defaultImgURL}'" alt="${name}" width="100" height="100" class=portrait>
       <div class="name">${name}</div>
       <div class="id">${id}</div>
-      <div class="budget">€${budget}</div>
```

Comment:
This is the correct version, so again, the merge conflict was resolved incorrectly. In case you're not sure why a certain conflict emerges, ask the person who wrote that piece of code that is conflicting with yours ;)